code/game/machinery/scanner_gate.dm

```diff
@@ -46,6 +46,10 @@
 	var/light_fail = FALSE
 	///Does the scanner ignore light_pass and light_fail for sending signals?
 	var/ignore_signals = FALSE
+	///Modifier to the chance of scanner being false positive
+	var/false_positive = 0
```

Comment:
this variable name is misleading. If this intends to be a reduction to the chance of a false positive, it needs to be made more clear.
The better thing to do might be to negate the value instead and then add it to the base_false_positive value when doing a prob check